projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-image-block-control/index.tsx

```diff
@@ -3,14 +3,19 @@
  */
 import { MediaUploadCheck, MediaUpload } from '@wordpress/block-editor';
 import { ToolbarButton, Dropdown, NavigableMenu, MenuItem } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { linkOff, image as imageIcon } from '@wordpress/icons';
-import { VideoControlProps } from '../../types';
-import styles from './style.module.scss';
 /**
  * Internal dependencies
  */
-import { VideoPosterCardProsp } from './types';
+import styles from './style.module.scss';
+/**
+ * Types
+ */
+import type { VideoPosterCardProsp } from './types';
```

Comment:
```suggestion
import type { VideoPosterCardProps } from './types';
```

This one needs to be applied at the types file as well.